### PR TITLE
Create acta-ornithologica.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/84555/style-request-acta-ornithologica

### CODE REVIEW

File: acta-ornithologica.csl [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+              <group delimiter=": ">
+                <text term="available at" text-case="capitalize-first"/>
+                <text variable="URL"/>
+              </group>
+            </group>
+          </else-if>
+          <else>
+            <group delimiter=". ">
+              <text macro="title"/>
+              <text variable="container-title" form="short" strip-periods="true"/>
```
Comment (adam3smith) [inline acta-ornithologica.csl:132]: I don't think this is right: The journal abbreviations are _with_ period but there is _no_ period after the journal name, see e.g. `Zool. Zhurnal 72: 85–97.` from the example article

**Comment by adam3smith** [comment]
One quick thing, looks good otherwise - thanks!

File: acta-ornithologica.csl (hunk 1 above)
Comment (POBrien333) [reply acta-ornithologica.csl]: Yeah, thanks for spotting that.
I looked at "Acta Ornithol." and didn't get that Acta is a full word...

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**